src/components/authentication/Login.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { loginUser } from '../../actions/authActions';
+
+class Login extends Component {
```

Comment:
You can add an export here as per the question you asked.
```suggestion
export class Login extends Component {
```

this will be the one you'll use for testing.

Reply:
And leave the export default as it is.

Reply:
okay, thanks.